### ISSUE TITLE
High Ping in some regions

### ISSUE THREAD

**Issue by waitwhatActo** [HomeworkSMP/server#2]
# Status: Monitoring
Description: Players in Asia, Africa, and Oceania may experience higher ping.
Known Affected Version: Java Asia, Java Africa, Java Oceania, Bedrock Asia, Bedrock Africa, Bedrock Oceania

**Comment by waitwhatActo** [comment]
A fix will be attempted between Jan 1st 00:00 UTC - Jan 5th 00:00 UTC. 
During this time, the server will operate as usual.

**Comment by waitwhatActo** [comment]
Would be great if we can get real player experience input for this issue.
cc @AvogatoWizardWhisker @VeronMDevstudio

**Comment by AvogatoWizardWhisker** [comment]
> Would be great if we can get real player experience input for this issue.
> cc @AvogatoWizardWhisker @VeronMDevstudio 

I was trying to joon the server twice but it returns to `getsocketopt`. I'll see if I'm available to test the server today.

**Comment by AvogatoWizardWhisker** [comment]
Currently, i'm experiencing 240ms latency but luckily it doesn't get me disconnected. That might be an improvement in my case because of I upgraded to Fiber.

**Comment by waitwhatActo** [comment]
# Issue Update
This issue is now under monitoring. ***Feel free to reopen if you encounter the same issue.***

## TL;DR
An improvement in ping connection between players and the server can be seen after the change in server hardware. While the results are conclusive, some players are now experiencing higher ping than usual.
